src/components/SongLibrary/SongLibrary.js

```diff
@@ -64,10 +64,10 @@ class SongLibrary extends Component {
     return (
       <>
         <SearchBar searchForSongs={this.searchForSongs} />
-        <h2 data-cy="search-msg">{this.state.searchResultsMsg}</h2>
-        <div className="library">
+        <p data-cy="search-msg">{this.state.searchResultsMsg}</p>
+        <main className="library">
           {allSongs}
-        </div>
+        </main>
```

Comment:
Glad that we don't get an error from the route-rendering, will implement this in future projects